docs/data/charts/sankey/sankey.md

```diff
@@ -107,15 +107,25 @@ To dynamically customize the order, use the sorting functions for the element th
 
 ### Node sorting
 
-The `nodeOptions.sort` function allows control of the vertical order of nodes within each column.
-It receives two `SankeyLayoutNode` objects and should return a number (similar to [`Array.sort()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#comparefn)).
+The `nodeOptions.sort` property controls the vertical order of nodes within each column.
+
+It accepts three types of values:
```

Comment:
Depending on how we look at it, there are only two types: a function or a string, so might be better to simplify as:

```suggestion
It accepts the following values:
```